fili-core/src/main/java/com/yahoo/bard/webservice/sql/helper/SqlAggregationType.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.sql.helper;
+
+import com.yahoo.bard.webservice.druid.model.aggregation.Aggregation;
+
+import org.apache.calcite.sql.SqlAggFunction;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+import org.apache.calcite.tools.RelBuilder;
+
+import java.util.Locale;
+
+/**
+ * All the aggregation types supported for use with a sql backend.
+ */
+public enum SqlAggregationType {
```

Comment:
Ok thanks for the suggestion! I've implemented this strategy, but haven't yet hooked it up in the BinderFactory (I'll do that in the next pull request)